src/alert.cpp

```diff
@@ -1463,16 +1463,18 @@ namespace {
 	}
 
 	peer_connect_alert::peer_connect_alert(aux::stack_allocator& alloc, torrent_handle h
-		, tcp::endpoint const& ep, peer_id const& peer_id, socket_type_t const type)
+		, tcp::endpoint const& ep, peer_id const& peer_id, socket_type_t const type, direction_t const dir)
 		: peer_alert(alloc, h, ep, peer_id)
 		, socket_type(type)
+		, direction(dir)
```

Comment:
this initialization list entry is not in the order of initialization: https://travis-ci.org/github/arvidn/libtorrent/jobs/692316456#L961